src/Payments/PlaidSDKIframe.res

```diff
@@ -46,6 +51,9 @@ let make = () => {
     if isReady && linkToken->String.length > 0 {
       let handler = Plaid.create({
         token: linkToken,
+        onLoad: _ => {
+          logger.setLogInfo(~value="Plaid SDK Loaded", ~eventName=PLAID_SDK_LOADED, ())
```

Comment:
```suggestion
          logger.setLogInfo(~value="Plaid SDK Loaded", ~eventName=PLAID_SDK, ())
```
So that other debug events can be logged with the same eventName but different messages in value